packages/preview/chuli-cv/0.1.0/lib.typ

```diff
@@ -0,0 +1,259 @@
+#import "@preview/fontawesome:0.1.0": *
+#import "modules/styles.typ": *
+#import "modules/header.typ": *
+#import "modules/section.typ": *
+#import "modules/skills.typ": *
+#import "modules/languages.typ": *
+#import "modules/piechart.typ": *
+#import "modules/education.typ": *
+#import "modules/entry.typ": *
+
+
+// Cover letter Components
+//--------------------------------------------
+
+#let letter_header(
```

Comment:
It is recommended to use kebab-case names. For instance, it would be `letter-header` here, but there are other places in the package where snake_case or camelCase is being used. This is however not mandatory, and if you prefer we can merge the package as it is.

Reply:
Thanks for the feedback! I've updated the commit to use kebab-case naming convention, as you recommended.